service/server.py

```diff
@@ -8,12 +7,12 @@
 import re
 import time
 
-from service import (address_utils, api_client, app, auditing, health_checker, login_api_client,
-                     login_manager, title_formatter, title_utils)
+from service import (address_utils, api_client, app, auditing, health_checker,
+                     title_formatter, title_utils)
 from service.forms import TitleSearchForm, SigninForm
 
+app.secret_key = 'F12Zr47j\3yX R~X@H!jmM]Lwf/,?KT'
```

Comment:
Why have we got a secret key in the server file? Shouldn't this be a environment variable?!